traversal/procedure/ProcedureEdge.java

```diff
@@ -970,7 +966,7 @@ private List<AttributeVertex<?>> branchToEq(
                     private List<Pair<TypeVertex, Forwardable<ThingVertex, Order.Asc>>> branchToTypes(
                             GraphManager graphMgr, ThingVertex owner
                     ) {
-                        Set<TypeVertex> types = graphMgr.schema().ownedAttributeTypes(owner.type());
+                        Set<TypeVertex> types = graphMgr.schema().ownedAttributeTypes(owner.type(), set());
```

Comment:
if this is meant to be an immutable empty set, we should use `emptySet()`, the singleton immutable empty set to avoid reallocation